theme/styles/snippets/product-info-bar.scss

```diff
@@ -14,9 +14,18 @@ $product-detail-height: 3.25rem;
   }
 
   &__product-details {
-
     @include media('md-up') {
       border-bottom: 0;
     }
   }
+
+  &__type {
+    &::after {
+      content: '';
+      display: block;
+      height: 0;
+      width: 0;
+      margin-bottom: (calc((1 - 1.13) * 0.5em));
+    }
+  }
```

Comment:
"Cropping" the line height to make it more visually centered vertically. This doesn't change the actual line height of the text but visually looks more centered next to the ●

[Ticket](https://www.notion.so/garden3d/ba130e56c01f4d9284d8bbf6cf59ba0f?v=133f51dd370c40ad981fef24024fa8b6&p=1dfe2e0fecfe444ebbeff8fa4e2d86fe)